Analysis/LeptonID/src/PIDutil.cc

```diff
@@ -0,0 +1,374 @@
+#include "PIDutil.h"
+#include <cmath>
+#include <set>
+#include <TF2.h>
+
+#include <marlin/VerbosityLevels.h>
+
+#include <EVENT/ReconstructedParticle.h>
+
+float getCorrectdEdx(float dEdx, Track *track)
+{
+    // dEdx correction stolen from the dEdx correction processor
+    double lambda = abs(atan(track->getTanLambda()) * 180.0 / M_PI);
+    std::vector<double> par = {0.970205, 0.0007506, 4.41781e-8, 5.8222e-8};
```

Comment:
Is there any way to re-use that existing functionality? Otherwise this is a prime candidate to get out of sync given the hardcoded values here.

Reply:
This can just be removed, it was in here for yet another processor that just made some n-tuples...